wgpu-core/src/binding_model.rs

```diff
@@ -474,7 +513,11 @@ impl<A: hal::Api> Resource for BindGroupLayout<A> {
 
     fn label(&self) -> &str {
         #[cfg(debug_assertions)]
-        return &self.label;
+        return if let BglOrDuplicate::Inner(ref inner) = self.inner {
+            &inner.label
+        } else {
+            ""
+        };
```

Comment:
nitpick:

```suggestion
        return self.as_inner().map_or("", |inner| &inner.label);
```

Pushed as a `fixup!` with 466309dc.